<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRadiologue</class>
 <widget class="QDialog" name="AddRadiologue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="nomEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>*ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="addRadiologueLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter personnel de soins</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>*Pr√©nom</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typePermicomboBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="idEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>*Type permi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>*Nom</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="prenomEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="OkButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
